Source/Analytics/Web/src/components/NationalSocietyOverview.js

```diff
@@ -41,7 +42,8 @@ class NationalSocietyOverview extends Component {
                     </Grid>
 
                     <HealthRiskPerDistrictTable />
-                    <Map />
+                    {/* <Map /> */}
+                    <ReportsPerHealthRiskPerDay></ReportsPerHealthRiskPerDay>
```

Comment:
As this component has no children, it should be self-closing: `<ReportsPerHealthRiskPerDay />`